apps/dsp-app/src/app/project/ontology/property-info/property-info.component.html

```diff
@@ -37,16 +37,29 @@
       [matTooltip]="'id: ' + propDef.id"
       matTooltipPosition="above"
       matTooltipClass="wide-tooltip"
-      class="mat-caption">
-      {{propDef.id | split: '#':1}}
+      class="mat-caption"
+      [matTooltipDisabled]="isLockHovered"
+      >> {{propDef.id | split: '#':1}}
```

Comment:
```suggestion
      [matTooltipDisabled]="isLockHovered">
      {{propDef.id | split: '#':1}}
```
One too many closing bracket. Didn't the linter spot is a an error?